packages/auth/src/AuthProvider.tsx

```diff
@@ -195,6 +195,17 @@ export const AuthProvider: React.FC<AuthProviderProps> = ({
 
   const getPassword = (phoneNumber: string) => MD5(`${phoneNumber}`).toString();
 
+  const signInUser = useCallback(
```

Comment:
this was remove on previous update of this repo but I realized we don't want adminweb to use signInOrCreate.